runtime/parser/parser_test.go

```diff
@@ -506,7 +506,7 @@ func TestParseBuffering(t *testing.T) {
 
           fun isneg(x: SignedFixedPoint): Bool {   /* I kinda forget what this is all about */
               return x                             /* but we probably need to figure it out */
-                     <                             /* ************/((TODO?{/*))************ *//
+                     <                             /* ************/((TODO?/*))************ *//
```

Comment:
Why was this change necessary?

Reply:
It's not strictly necessary, but the presence of the `{` here makes the parser think that the reason this failed is because it was trying to parse a restricted type. The test immediately after this keeps the `{` and changes its error message to be the restricted type parsing failure message, so here I chose to remove the `{` to test both behaviors.